### PR TITLE
TDP-4169 - Add documentation that the text body is required.

### PR DESCRIPTION
**Pull request by mrmeyers99**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4169-empty-speak-sentence.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/speakSentence.md [hunk 1]
```diff
@@ -19,6 +19,13 @@ To choose a specific voice by name, use the `voice` attribute.
 | gender    | Selects the gender of the speaker. Valid values are `"male"` or `"female"`.<br><br>Default `"female"`                                                                            |
 | locale    | Selects the locale of the speaker. Consult `locale` column in the below table for valid values.<br><br>Default `"en_US"`                                                         |
 
+
+### Text Content
+| Name     | Description                                                                                                                                  |
+|:---------|:---------------------------------------------------------------------------------------------------------------------------------------------|
+| text | The text to speak. Cannot be blank. Can be a mixture of plain text and SSML tags (see below for list of supported tags). |
```
Comment (n-miles) [inline voice/bxml/verbs/speakSentence.md:26]: So this is formatted as a table? Seems kind of awkward. Not sure how else to express it though...

Comment (jmulford-bw) [reply voice/bxml/verbs/speakSentence.md:26]: It is. You don't have to align the `|`

http://bw-dev-tdp-4169-empty-speak-sentence.s3-website-us-east-1.amazonaws.com/voice/bxml/verbs/speakSentence.html is what it looks like on the page

Comment (jmulford-bw) [reply voice/bxml/verbs/speakSentence.md:26]: Actually I think I misread your question. It's 2 separate tables if that's what you were asking

Comment (n-miles) [reply voice/bxml/verbs/speakSentence.md:26]: Yeah it looks good rendered. Just seems odd to me to make it a table when there's only one element in it

Comment (mrmeyers99) [reply voice/bxml/verbs/speakSentence.md:26]: I did it to mimick the play audio page https://dev.bandwidth.com/voice/bxml/verbs/playAudio.html

Comment (scapps-bw) [reply voice/bxml/verbs/speakSentence.md:26]: I only like this if the voice table gets its own header. Right now it looks like the voice table is part of the "Text Content" section.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4169-empty-speak-sentence.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/speakSentence.md (hunk 1 above)
Comment (mrmeyers99) [reply voice/bxml/verbs/speakSentence.md:26]: updated

**Review by n-miles — APPROVED** [review]

**Review by scapps-bw — APPROVED** [review]

**Pull request merged**